packages/api-explorer/src/Doc.jsx

```diff
@@ -106,9 +106,13 @@ class Doc extends React.Component {
     this.setState({ showEndpoint: true });
   }
 
-  renderContentTypeSelect(){
+  renderContentTypeSelect() {
     const list = getContentTypeFromOperation(this.getOperation())
-    return <Select options={list} onChange={(e) => this.setState({selectedContentType: e.currentTarget.value})} />
+    return (<Select
+      value={this.state.selectedContentType}
+      options={list}
+      onChange={(e) => this.setState({selectedContentType: e})} 
```

Comment:
If e is the value of the select, and not the event, we should change params name